app/players/elves/elf.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC
+from app.players.player import Player
```

Comment:
Do not forget to separate different groups of imports with a blank line according to PEP-8.